enzy_htp/mutation/api.py

```diff
@@ -367,8 +369,59 @@ def mutate_stru_with_tleap(
 
     return stru_cpy
 
+def mutate_stru_with_pymol(
+        stru: Structure,
+        mutant: List[Mutation],
+        in_place: bool = False,
+        ) -> Structure:
```

Comment:
The path of the intermediate file could also be an special keyword arg to this function. (Reference int_leapout_pdb_path mutate_stru_with_tleap())